editor/js/Sidebar.Material.js

```diff
@@ -764,6 +764,15 @@ function SidebarMaterial( editor ) {
 				if ( objectHasUvs ) {
 
 					var map = mapEnabled ? materialMap.getValue() : null;
+
+					// Add the texture to the editors textures map
+					if ( materialMap.getValue() !== null ) {
+
+						const texture = materialMap.getValue();
+						editor.addTexture( texture );
+
+					}
```

Comment:
I think we can add `if ( texture === null ) return;` [here](https://github.com/mrdoob/three.js/blob/9c79e50e5e5163d9d4db18838d3cbd3e05c6efc1/editor/js/Editor.js#L392) and we won't need to all these checks here.